key-concepts/covenants.adoc

```diff
@@ -1,11 +1,174 @@
 = Covenants
 
-A covenant is a set of link:actions.adoc[action] creators and types, a
-reducer (which is a link:smart_contracts.adoc[smart contract], and a
-Redux link:sagas.adoc[saga] that together indicate the functioning of an
-Interbit blockchain. These actions, smart contract, and saga work
-together to fulfill the functionality of a single Interbit blockchain.
-The `rootSaga` in a smart contract is optional.
+[quote,Merriam Webster Dictionary]
+**covenant**: a written agreement or promise, usually under seal,
+between two or more parties, especially for the performance of some
+action.
+
+Interbit blockchain networks are governed by covenants. Each node in the
```

Comment:
~blockchain networks~ blockchains

This paragraph is ambiguous. I would say the blockchain is governed by consensus as the consensus determines whether the covenant actually gets applied. The network itself, comprised of many chains and nodes, isn't governed by just covenants, but also joins between chains. 

A blockchain network may be comprised of many blockchains all running different covenants. A node may run many blockchains running many covenants. One blockchain runs one covenant on many nodes within a blockchain network.

The single blockchain's consensus mechanism is the force that decides whether or not to apply the covenant hash to the blockchain and thus which code to run on the chain.